packages/grafana-ui/src/components/Table/SVGCell.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { DisplayValue, Field } from '@grafana/data';
+import { TableStyles } from './styles';
+import { TableCellProps } from './types';
+
+export const SVGCell: FC<TableCellProps> = props => {
+  const { field, cell, tableStyles, cellProps } = props;
+
+  const displayValue = field.display!(cell.value);
+
+  const cellStyle = getCellStyle(tableStyles, field, displayValue);
+
+  return (
+    <div
+      {...cellProps}
+      className={cellStyle + ' ' + tableStyles.imageCell}
+      dangerouslySetInnerHTML={{ __html: displayValue.text }}
```

Comment:
we cannot allow this, not secure

Reply:
I could add safe-html (https://www.npmjs.com/package/safe-html) to ensure that is only a SVG.

Ok?

Reply:
we already have sanitize functions in grafana/data https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/text/sanitize.ts

Reply:
Jums... I see that configuration is only for CSS attributes. So, I need create a new config por XSS with some tags.

I think... a, span, strong, italic, svg, img, h1-6.... any else?